app/parsers/stackdriver.py

```diff
@@ -13,43 +13,102 @@
 # limitations under the License.
 
 
+import dateutil.parser
 import jmespath
+import os
+import time
+from rpe.resources.gcp import GoogleAPIResource
+
+from .base import ParsedMessage
 
 
 class StackdriverParser():
     ''' A collection of functions for parsing Stackdriver log messages '''
 
     @classmethod
-    def _is_auditlog(cls, message):
-        ''' Check whether or not a log message is an audit log '''
-        audit_log_type = 'type.googleapis.com/google.cloud.audit.AuditLog'
-        message_type = jmespath.search('protoPayload."@type"', message)
-        return message_type == audit_log_type
+    def _delay(cls, message_age):
```

Comment:
currently, I'd expect `ENFORCEMENT_DELAY` to affect all of the triggers, not just stackdriver.  Is it in this parser for a reason?

Reply:
The event-driven triggers make sense to delay. The idea was to make sure enforcement didn't cause weird errors for tools like terraform.

I toyed with the idea of providing a way for message parsers to have some control over the enforcer. I'll rework this to allow message parsers to toggle enforcement, and whether or not to delay.